src/models/multiChart.js

```diff
@@ -95,7 +95,7 @@ nv.models.multiChart = function() {
                     })
                 });
 
-            x   .domain(d3.extent(d3.merge(series1.concat(series2)), function(d) { return getX(d) }))
+            x   .domain(d3.extent(d3.merge(series1.concat(series2)), function(d) { return d.x }))
```

Comment:
I don't think this is right, the x value should always bet retrieved via getX because people can override that to get the x value from a differently named attribute.